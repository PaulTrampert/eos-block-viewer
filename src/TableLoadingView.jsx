import React from 'react';
import Spinner from 'react-bootstrap/Spinner';
import styles from './TableLoadingView.module.css';

const TableLoadingView = () => {
  return (
    <tr>
      <td colSpan={3} className={styles.loading}>
        <Spinner animation="border" variant="primary" />
      </td>
    </tr>
  );
}

export default TableLoadingView;