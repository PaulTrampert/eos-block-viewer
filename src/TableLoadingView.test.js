import React from 'react';
import {shallow} from 'enzyme';
import TableLoadingView from './TableLoadingView';

describe('TableLoadingView', () => {
  let subject;

  beforeEach(() => {
    subject = shallow(<TableLoadingView />);
  });

  it('renders a Spinner in the table row', () => {
    expect(subject).toMatchSnapshot();
  });
});