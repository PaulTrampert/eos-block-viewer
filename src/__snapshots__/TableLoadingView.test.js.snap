// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableLoadingView renders a Spinner in the table row 1`] = `
<tr>
  <td
    className="loading"
    colSpan={3}
  >
    <Spinner
      animation="border"
      variant="primary"
    />
  </td>
</tr>
`;
